src/sentry/static/sentry/app/components/platformList.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import PlatformIcon from 'app/components/platformIcon';
+import {PlatformKey} from 'app/data/platformCategories';
+
+type Props = {
+  className?: string;
+  platforms?: PlatformKey[];
+  direction?: 'right' | 'left';
+  /**
+   * Maximum number of platform icons to display
+   */
+  max?: number;
+  /**
+   * Platform icon size in pixels
+   */
+  size?: number;
+  /**
+   * Will set container width to be size of having `this.props.max` icons
+   * This is good for lists where the project name is displayed
+   */
+  consistentWidth?: boolean;
+};
+
+const PlatformList = ({
+  platforms = [],
```

Comment:
I noticed that we usually use defaultProps to define the default Props :)

Reply:
This is a function component, in which the ES6 defaults are preferred.

Reply:
yes, but if you check out some of our function components, for example, `alert.jsx`, you will see that defaultProps is being used...now I am confused 🤯

Reply:
defaultProps on functions will eventually get deprecated, see https://develop.sentry.dev/frontend/defaultprops/#function-components